tests/console/system_prepare.pm

```diff
@@ -34,8 +34,12 @@ sub run {
 
     prepare_serial_console;
 
-    # Make sure packagekit is not running, or it will conflict with SUSEConnect.
-    pkcon_quit unless check_var('DESKTOP', 'textmode');
+    if (!check_var('DESKTOP', 'textmode')) {
+        # Make sure packagekit is not running, or it will conflict with SUSEConnect.
+        pkcon_quit unless check_var('DESKTOP', 'textmode');
+        # poo#77134 wait for packagekit related zypper process to exit
+        wait_still_screen 10;
```

Comment:
this is unreliable and should be avoided. If you need to wait for the related zypper processes then do exactly that, e.g.:

```
assert_script_run 'until ! pgrep zypper; do sleep 1; done';
```

Reply:
Yes, this is a better way to wait the block zypper process.